lib/rules/one-var.js

```diff
@@ -272,6 +276,87 @@ module.exports = {
             return true;
         }
 
+        /**
+         * Fixer to join VariableDeclaration's into a single declaration
+         * @param   {VariableDeclaration} declarations The `VariableDeclaration` to join
+         * @returns {Function}                         The fixer function
+         */
+        function joinDeclarations(declarations) {
+            const declaration = declarations[0];
+
+            return fixer => {
+                const type = sourceCode.getTokenBefore(declaration);
+                const prevSemi = sourceCode.getTokenBefore(type);
+                const prevType = sourceCode.getTokenBefore(type, {
+                    filter(token) {
+                        return ~["const", "var", "let"].indexOf(token.value) && token.type === "Keyword";
+                    }
+                });
+                const res = [];
+
+                if (sourceCode.getText(prevType) === sourceCode.getText(type)) {
+                    res.push(fixer.replaceText(prevSemi, ","));
+                    res.push(fixer.replaceText(type, ""));
+                }
+
+                return res;
+            };
+        }
+
+        /**
+         * Fixer to split a VariableDeclaration into individual declarations
+         * @param   {VariableDeclaration}   declaration The `VariableDeclaration` to split
+         * @param   {?Function}             filter      Function to filter the declarations
+         * @returns {Function}                          The fixer function
+         */
+        function splitDeclarations(declaration) {
+            return fixer => declaration.declarations.map(declarator => {
+                const comma = sourceCode.getTokenAfter(declarator);
+                const afterComma = sourceCode.getTokenAfter(comma, { includeComments: true });
+
+                if (comma.value === ",") {
```

Comment:
I'm confused by this variable name. When would the value not be `,`? If the token is not always a comma, can you give it a more descriptive name (e.g. `tokenAfterDeclarator`)?